BeVolt/App/inc/Images.h

```diff
@@ -0,0 +1,2 @@
+extern const char *utsvt_logo;
+extern const char *party_parrot_frames[10];
```

Comment:
How would I go about doing that?

Reply:
Basically, I think an `#ifdef` guard around everything that has to do with the party parrot would be a good idea. The definition could come from the terminal. for example, if the guard was `#ifdef PARTY_PARROT`, then building the target with the `-DPARTY_PARROT` argument on the cmd line would enable the feature, but otherwise, it wouldn't be included in the compiled binary.